uosc.lua

```diff
@@ -3149,6 +3149,35 @@ end
 -- **active_path** - full path of a file to preselect
 -- Rest of the options are passed to `menu:open()`
 function open_file_navigation_menu(_directory, handle_select, menu_options)
+	local items = {}
+
+	if _directory == '$drives' then
```

Comment:
This should be a separate function `open_drives_menu()`.

Reply:
OK, no problem, that's correct!